### ISSUE TITLE
How to use InputBox in xlwings to prompt the user for a range and retrieve cell contents interactively (or an alternative) in Python?

### ISSUE THREAD

**Issue by user13391015** [xlwings/xlwings#1909]
#### [Related to this SO question](https://stackoverflow.com/questions/72102955/how-to-use-inputbox-in-xlwings-to-prompt-the-user-for-a-range-and-retrieve-cell)

During the execution of the Python notebook I want to prompt the user for a range in the Excel workbook in order to retrieve the cell contents interactively (for further elaboration in pandas, which I use to overcome limitations in Excel). An user-friendly (point-and-click) example to achieve that in VBA would be as follows:

```VBA
Dim LoopR As Range: Set LoopR = Application.InputBox("Select the range (rows) to iterate over:", "Iteration Range Rows Input", Type:=8)

```
Which creates a dialog box where the user can select the range manually, and returns it.

I attempted to achieve the same in xlwings with the following:

```python
import xlwings as xw
app = xw.App(add_book=False)
wb = app.books.open('path/to/file.xlsx')
rng = app.api.InputBox("Select the range (rows) to iterate over:", "Iteration Range Rows Input", Type=8)

```
Which creates the dialog box in the Excel application as expected; however, `rng` only stores the content (as string) of the top-left cell of the selected range, which in my case defeats the purpose. I haven't found any threads about my specific issue and wouldn't know how to achieve the same using the win23api module.

I think it's possible to achieve what I want in xlwings without having to awkwardly input the range by typing or hardcoding it. Please note that I can't upgrade xlwings (my version is 0.16.0) and therefore can't use the convenient [`xw.load()`](https://docs.xlwings.org/en/stable/jupyternotebooks.html#the-load-function) function.

I welcome solutions, explanations, or valid alternatives.

**Comment by fzumstein** [comment]
I can't replicate this:

```python
import xlwings as xw
wb = xw.books.active
rng = wb.app.api.InputBox("Select the range (rows) to iterate over:",
                          "Iteration Range Rows Input", Type=8)
print(rng.Address)
```

prints:

```
$A$1:$D$9
```

**Comment by fzumstein** [comment]
you're using pywin32 here directly, so maybe try to upgrade/reinstall it

**Comment by maksim-pinguin** [comment]
Tl;dr;
Use list args instead of kwargs
```
type=8
InputBox('msg',None,None,None,None,None,None,type)
```
Was getting the same error with pure win32com. Seams like a of-by-one bug. The keyword argument 'Type' is ignored. 

To reproduce do:
```
Inputbox(**{'Prompt':'msg', 'Type':8})
```

python3.10 
pywin32 305
Windows 10
